git-index/src/lib.rs

```diff
@@ -22,6 +22,10 @@ mod access {
         pub fn entries(&self) -> &[Entry] {
```

Comment:
The lib should start out with: `#![deny(unsafe_code, rust_2018_idioms)]`.

Reply:
Doesn't it have one here: https://github.com/Byron/gitoxide/blob/91c064e48692f25f04a3d1ae8dacb7056d307bda/git-index/src/lib.rs#L1 ?

Reply:
Sorry, I was looking at `git-index` all the time 🤦‍♂️.